tests/constants.py

```diff
@@ -17,3 +17,9 @@
 
 RINDEX_ENTRY_SIZE = 64
 RINDEX_ENTRIES_SECTORS = 2000
+
+# A few file/disk size constants.
+KiB = 1024      # 1 Kibibyte = 2^10 bytes = 1024 bytes
+MiB = 1024**2   # 1 Mebibyte = 2^20 bytes = 1,048,576 bytes = 1024 kibibytes
+GiB = 1024**3   # 1 Gibibyte = 2^30 bytes = 1,073,741,824 bytes = 1024 mebibytes
+TiB = 1024**4   # 1 Tebibyte = 2^40 bytes = 1,099,511,627,776 bytes = 1024 gibibytes
```

Comment:
Looks good, but I don't we need the comments about the values.

I think it would be nicer to add the units in a new units.py module - since we really want to write code like:

    from units import *
    ...
    size = 2 * GiB

Interesting that there is a patch in qemu doing the same thing:
http://lists.nongnu.org/archive/html/qemu-block/2019-05/msg00425.html